test/highlight/comments.scala

```diff
@@ -2,3 +2,11 @@
 //  ^keyword
 //        ^parameter
 //              ^string
+
+/*
+ * Beep boop
+ */
+// <- comment
```

Comment:
Nice, was just about to ask but then [read the docs](https://tree-sitter.github.io/tree-sitter/syntax-highlighting#unit-testing):

> Arrow: <- this will test the following selector against the scope on the most recent non-test line. It will test it at the same column as the comment character is in.